gen3/bin/kube-setup-dicom.sh

```diff
@@ -82,6 +82,21 @@ EOM
 }
 EOM
   fi
+
+  if [[ ! -f "$secretsFolder/app-config.js" ]]; then
```

Comment:
The `app-config.js` in this repo will only be used for the 1st deployment. Instead should we ensure that every time we run `kube-setup-dicom`, we pull the config from cloud-automation? This way we can maintain the config here instead of editing directly in the vm?

Reply:
I can remove the check, so every time it will create from the latest cloud-automation.